src/modal/modal-config.ts

```diff
@@ -62,6 +62,12 @@ export interface NgbModalOptions {
    * @since 1.1.0
    */
   backdropClass?: string;
+
+  /**
+   * Assign focus to the first focusable element when opened (true by default)
+   * @since 3.3.0
```

Comment:
Please don't add the version since we don't know when it is going to be merged. I will add those when cutting a release.

Reply:
Ok, I will remove it !